app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -362,11 +407,8 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler, AppStateLis
         }
     }
 
-    private fun showWhatsNewIfNeeded() {
```

Comment:
Why not keep this check in one place? Maybe with an optional flag to check the lifecycle? Or put the lifecycle check around  `showWhatsNewIfNeeded()`

Reply:
it won't work if passcode is not required. In this case activity is not yet resumed, but we still need to show the dialog